BombersRTT/Assets/Framework/BaseManagers/GPlayerMgr.cs

```diff
@@ -712,21 +712,6 @@ public long GetCurrencyBalance(string in_type)
             }
             return toReturn;
         }
-
-        public void ValidateString(string in_string, SuccessCallback success = null, FailureCallback failure = null)
-        {
-            if (in_string.Length < MIN_CHARACTERS_GAME_NAME)
```

Comment:
You shouldn't remove the ValidateString method completely as some other checks still need to be made to validate a string like in this case, the string needs to be a certain length.  I would only remove the call to braincloud.

Reply:
If the condition is successful, I would then call the m_validateStringSuccess callback.